FirebaseSwiftUI/FirebaseAuthSwiftUI/Sources/Views/MFAEnrolmentView.swift

```diff
@@ -87,11 +85,10 @@ public struct MFAEnrolmentView {
           issuer: authService.configuration.mfaIssuer
         )
         currentSession = session
+        isLoading = false
       } catch {
-        errorMessage = error.localizedDescription
+        isLoading = false
```

Comment:
defer { isLoading = false } outside of do/catch block if you want isLoading to always reset to false